README.md

```diff
@@ -151,3 +151,25 @@ advanced building strategies and other development notes.
 
 There is PyDoc generated documentation available for the VPP test framework.
 See @ref test_framework_doc for details.
+
+
+
+### Build and Upload Docker VPP Image 
+
+Build and Upload VPP Docker Image comprise of three parts: 
+
+1. GitHub :  https://github.com/travelping/vpp    # This repository is used for version control of VPP code.
+2. GitLab :  https://gitlab.com/travelping/vpp-github  #  This repository is configured as mirror repository with upstream to GitHub repository. This part is used for CI/CD pipeline only.
+3. DockerHubRegistry: https://hub.docker.com/u/ergw/dashboard/    # After successful build new VPP Docker Image will be uploaded to this registry. 
+
+
+Build and Upload Pipeline. 
+
+1. Commit on https://github.com/travelping/vpp repository will be mirrored to the https://gitlab.com/travelping/vpp-github and triger CI/CD PipeLine on GitLab. Mirroring (Sync) between GitHub and GitLab occurs automatically every 30 minutes or can be trioggered manually. 
+2. GitLab CI/CD will trigger build of Docker Image, build proccess will be executed on GtiLab-Runner (Agent). GtiLab-Runner is running on "hamster" K8S cluster. Use the following URL https://gitlab.com/groups/travelping/-/settings/ci_cd to see available GitLab-Runner.
```

Comment:
Don't expose "hamster". You can state that it is build on dedicated runner instances. You also don't need to refer to those settings. This is documentation at github.com, hence for public and not just for internal.